build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1530,6 +1530,11 @@ message DigestFunction {
 
     // The SHA-512 digest function.
     SHA512 = 6;
+
+    // Murmur3 128-bit digest function, x64/64-bit variant. Note that this is not a
```

Comment:
"x64/64-bit" is a bit rendundant, and might be slightly confusing given the "128-bit" in the first part of the sentence. Maybe just say "x64 variant" ?

Reply:
Fixed